.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Is this supposed to commented out per my request?

Reply:
Yes, as we discussed, I have created a new PR, if that wasn't commented it would run prettier on my local machine and change files. Now, you can play with configurations and test when uncommenting.

Reply:
It doesn't look commented out

Reply:
Sorry, my bad